eth/handler.go

```diff
@@ -320,26 +320,22 @@ func newHandler(config *handlerConfig) (*handler, error) {
 	}
 
 	broadcastBlockWithCheck := func(block *types.Block, propagate bool) {
-		// All the block fetcher activities should be disabled
```

Comment:
> may need to retain these codes? because it is used in propagate & not propagate block scenarios?

no
becaused these code have already exsited in the lamda function `validator`, it will be called before broadcastBlockWithCheck